pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -138,8 +153,5 @@ func SetupPodNetworkPhase2(vmi *v1.VirtualMachineInstance, domain *api.Domain) e
 }
 
 func newpodNIC(network *v1.Network) (podNIC, error) {
-	if network.Pod != nil || network.Multus != nil {
-		return new(podNICImpl), nil
-	}
-	return nil, fmt.Errorf("Network not implemented")
+	return new(podNICImpl), nil
```

Comment:
No need for this function now, right? It seems the next commit removed the unused argument, but this change already makes the function redundant (wrapping `new`).

Reply:
I'm afraid the tests, in their pre- @qinqon form, needs it.